hi-ml/testhiml/testhiml/test_run_ml.py

```diff
@@ -350,22 +348,7 @@ def test_log_on_vm(log_from_vm: bool) -> None:
     assert runner.trainer.loggers is not None
     assert len(runner.trainer.loggers) > 1
     logger = runner.trainer.loggers[1]
-    assert isinstance(logger, AzureMLLogger)
-    if log_from_vm:
```

Comment:
what happened to all these checks?

Reply:
There is no more native AzureMLLogger, and we don't have any of these properties in the new MLFlow logger so this functionality will need to be recreated for the new logger (in a separate PR)